Earthfile

```diff
@@ -1,11 +1,12 @@
-VERSION 0.6
-FROM node:lts-alpine
+VERSION --wait-block 0.6
```

Comment:
Do we need `--wait-block`?

Reply:
Nope. I tried to use `WAIT` at one point but we can drop this now.